core/src/main/java/org/sonarsource/sonarlint/core/hotspot/HotspotServiceImpl.java

```diff
@@ -56,6 +60,8 @@ public void openHotspotInBrowser(OpenHotspotInBrowserParams params) {
     var url = buildHotspotUrl(effectiveBinding.get().getSonarProjectKey(), params.getBranch(), params.getHotspotKey(), endpointParams.get());
 
     client.openUrlInBrowser(new OpenUrlInBrowserParams(url));
+
+    telemetryService.hotspotOpenedInBrowser();
```

Comment:
I think it's ok for now, maybe at some point we will need events to decouple services from telemetry

Reply:
yes, possibly